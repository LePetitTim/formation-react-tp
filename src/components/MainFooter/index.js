import React from 'react';

export const MainFooter = () => <p>2018 Makina Corpus</p>;

export default MainFooter;
