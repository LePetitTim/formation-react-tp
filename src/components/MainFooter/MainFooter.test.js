import React from 'react';
import renderer from 'react-test-renderer';
import MainFooter from './';

it('should render', () => {
  const tree = renderer.create(<MainFooter />).toJSON();
  expect(tree).toMatchSnapshot();
});
