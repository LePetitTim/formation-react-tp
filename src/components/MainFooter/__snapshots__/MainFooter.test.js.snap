// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<p>
  2018 Makina Corpus
</p>
`;
